inc/class-upgrade.php

```diff
@@ -814,6 +814,9 @@ private function upgrade_1591() {
 	 */
 	private function upgrade_163() {
 		$this->migrate_og_settings_from_social_to_titles();
+
+		// Run after the WPSEO_Options::enrich_defaults method which has priority 99.
+		\add_action( 'init', [ $this, 'set_og_settings_from_seo_values' ], 100 );
```

Comment:
Why not use `\PHP_INT_MAX`? Just to be safe...

Reply:
I think it's better to leave some "room" for hooking into `init` after the migration has been done. We need just to execute that after `enrich_defaults`, any other post type or taxonomy which is being registered after priority 99 won't get an option in our framework so the related `isset` checks in `set_og_settings_from_seo_values` would just fail.